src/main/com/fulcrologic/statecharts/data_model/working_memory_data_model.cljc

```diff
@@ -156,7 +156,7 @@
                 (if (map? data)
                   (do
                     (log/trace "Loaded" data "into context" state-id)
-                    (vswap! vwmem ::data-model assoc state-id data))
+                    (vswap! vwmem assoc ::data-model data))
```

Comment:
Um, is this right? Why not assoc-in like before?

Reply:
Oh wait...right, for the flat model it isn't supposed to have anything to do with the state in question.

Reply:
But my testing comment does beg the question: Should this be a merge? Should the load operation have such an option, even?